generators/java/sdk/versions.yml

```diff
@@ -1,3 +1,10 @@
+- changelogEntry:
+    - summary: |
+        Implement global timeout configuration for generated clients.
```

Comment:
nit: can you add a code snippet of how someone would override the timeout: 

```java
var client = Client.builder()
  .apiKey("...")
  .timeout(40)
  .build();
```

Reply:
Done